src/main/java/org/folio/search/service/LanguageConfigService.java

```diff
@@ -0,0 +1,65 @@
+package org.folio.search.service;
+
+import static org.folio.search.converter.LanguageConfigConverter.toLanguageConfig;
+import static org.folio.search.converter.LanguageConfigConverter.toLanguageConfigEntity;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.search.converter.LanguageConfigConverter;
+import org.folio.search.domain.dto.LanguageConfig;
+import org.folio.search.domain.dto.LanguageConfigs;
+import org.folio.search.exception.ValidationException;
+import org.folio.search.model.config.LanguageConfigEntity;
+import org.folio.search.repository.LanguageConfigRepository;
+import org.folio.search.service.metadata.ResourceDescriptionService;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+@AllArgsConstructor
+public class LanguageConfigService {
```

Comment:
No unit tests for this class

Reply:
Added, some of integration tests were removed and replaced with unit tests.